packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/components",
-	"version": "2.2.0",
+	"version": "2.3.0",
```

Comment:
Can we leave the actual release for another PR? I read your rationale:

> We bump the package version to 2.3.0 just in case something imports the package by version and we push the changes to NPM.

but I wasn't sure how this could cause problems, unless someone deletes and republishes `v2.2.0`.

Reply:
That's exactly what I meant - by merging this PR without the version change, you don't change `v2.2.0` in npm, unless you specifically republish it afterwards. And I don't think someone will just go ahead and republish an already published version. With that in mind, I don't see how this could happen:

> . Someone could technically be using number-formatters from version 2.2.0 and refresh their dependencies to find that component missing.

Reply:
> And I don't think someone will just go ahead and republish an already published version.

Haha indeed

Reply:
Also, the new release would also include other changes, so I agree that it's better to handle it separately.

Reply:
Thanks again, should be addressed now